sdk/communication/communication-network-traversal/README.md

```diff
@@ -0,0 +1,117 @@
+# Azure Communication Network Traversal client library for JavaScript
+
+Azure Communication Network Traversal is managing tokens for Azure Communication Services.
```

Comment:
This description is a little generic, doesn't mention TURN etc.

Reply:
If this file is a placeholder while awaiting language clarifications etc, i'd mention that at the top.
Otherwise, this needs to be tidied up